src/core/mempool.rs

```diff
@@ -917,7 +916,7 @@ impl MemPoolDB {
             &u64_to_sql(origin_nonce)?,
             &sponsor_address.to_string(),
             &u64_to_sql(sponsor_nonce)?,
-            &u64_to_sql(estimated_fee)?,
+            &u64_to_sql(tx_fee)?,
```

Comment:
Since this PR is headed to `develop` anyway, and `develop` already breaks the DB schema, can we just drop the `estimated_fee` column from the database?